Pong.xcodeproj/project.pbxproj

```diff
@@ -78,6 +90,19 @@
 			name = Frameworks;
 			sourceTree = "<group>";
 		};
+		09B3C6D627C2748B008DDEAE /* pong */ = {
+			isa = PBXGroup;
+			children = (
+				09B3C70C27C2AF6F008DDEAE /* p_game.c */,
+				09B3C71A27C2B2FE008DDEAE /* p_renderer.c */,
+				09B3C71C27C2B357008DDEAE /* p_scene.c */,
+				09B3C71E27C2B38E008DDEAE /* p_subsystems.c */,
+				09B3C72027C2B3B1008DDEAE /* p_window.c */,
+				09B3C6FE27C29B62008DDEAE /* pong.h */,
+			);
+			path = pong;
+			sourceTree = "<group>";
+		};
```

Comment:
[L] This file and function structure is inspired by Eskil Steenberg's approach to writing C modules under the his Quel Solaar grand project.

All of the files in general have a possibility of interdependency which is simply resolved using one consolidated header file, where applicable, while the rest of the implementation detail is distributed across multiple files of separate concerns.

However, there is also overengineering here to some extent. Some of the files carry very simple implementation detail that doesn't need to be repackaged from SDL, but I'll find out in the future whether that's all worth it.